pkg/rtc/rtc_linux_test.go

```diff
@@ -0,0 +1,43 @@
+// Copyright 2021 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package rtc
+
+import (
+	"os"
+	"testing"
+	"time"
+)
+
+func TestSet(t *testing.T) {
+	rtc, err := OpenMockRTC()
+	if err != nil {
+		t.Errorf("Error opening RTC: %v", err)
+	}
+	if err := rtc.Set(time.Now()); err != nil {
+		t.Errorf("Expected: nil Got: %v", err)
+	}
+	if err := rtc.Close(); err != nil {
+		t.Errorf("Failed to close RTC: %v", err)
+	}
+	if err := os.Remove(rtc.file.Name()); err != nil {
```

Comment:
If the mock creates the file under `t.TempDir()`, then it would be auto-deleted and none of these `os.Remove` lines are needed.